src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    private int radius;
+
+    public Circle(int radius) {
+        this.radius = radius;
+    }
+
+    @Override
+    public Figure createRandomFigure() {
+        return new Circle(getRandomInt());
+    }
+
+    @Override
+    public void printInformation() {
+        System.out.println("Circle radius: " + radius
+                + " area: " + (Math.PI * Math.pow(radius, 2))
+                + " Color " + getColor());
```

Comment:
Consider using a local variable to store the calculated area before printing it out, this will improve the readability of your code.